Sucuri/sucuri-wordpress-plugin
# Fixed password reset email link protocol

This PR fixes the issue whereby the password reset email included `http` links.

### How to test
- Pull the branch
- $ npm install
- Start the wp-env instance `$ npx npx wp-env start`
- Install and configure an SMPT plugin to handle emails locally `$ npx wp-env run ${container} "wp plugin install wp-mail-smtp --activate"`
- Create a test user `$ npx wp-env run ${container} "wp user create tester tester@example.com --role=author --user_pass=password"`
- Load http://localhost:8888/wp-admin/ and login with the wp-env admin user (user: admin, password: password)
- Navigate to http://localhost:8888/wp-admin/admin.php?page=sucuriscan_settings#posthack
- Under 'Reset User Password' select the test user and click submit.
- The password reset email link should have the `https` protocol
